.vscode/launch.json

```diff
@@ -8,7 +8,7 @@
       "request": "launch",
       "platform": "ios",
       "iosRelativeProjectPath": "Example",
-      "target": "iPhone 5s",
+      "target": "iPhone 6",
```

Comment:
Couldn’t actually test if this worked atm, because I’m being blocked by https://github.com/Microsoft/vscode/issues/10148#issuecomment-242315672. It’s fixed in VS Code master, so maybe I’ll try tomorrow’s ‘Insider’ build.